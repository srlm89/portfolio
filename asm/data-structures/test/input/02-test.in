ERASE actual/02-test.out
INSERT 0 3.483658 Chocolate 720p
PRINT_IN_ORDER actual/02-test.out
INSERT 3 0.955509 James Bond 12 - For Your Eyes Only [1981]
INSERT 4 4.721663 007 Licence to Kill [1989] [BluRay] [720p] [DTS]
REMOVE_ID 4
PRINT_IN_ORDER actual/02-test.out
