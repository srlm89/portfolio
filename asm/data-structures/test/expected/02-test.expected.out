[ 0 3.483658 Chocolate 720p ]
[ 0 3.483658 Chocolate 720p ][ 3 0.955509 James Bond 12 - For Your Eyes Only [1981] ]
